release-notes/1.0/1.0.19189.1-Preview.md

```diff
@@ -0,0 +1,159 @@
+
+.NET Framework
+
+* Fixed an issue when a pooled connection was switched between databases supporting UTF-8 which resulted in a mismatch between encoding and code page.
+
+.NET Core
+
+* When doing a hard copy of a sql parameter, the MetaType was not copied if the value of the parameter was not defined. This resulted in the wrong SqlType of the parameter.
+* Fixed in 1.0.19123.2-Preview
+  * XEvents are arriving as expected, but with 1 XEvent delay [CoreFx PR 36015](https://github.com/dotnet/corefx/pull/36015)
+  * Fix passing null for credentials in SqlConnectionPoolKey [CoreFx PR 34268](https://github.com/dotnet/corefx/pull/34268)
+  * Potential race condition in SqlConnectionString.cs [CoreFx Issue 30659](https://github.com/dotnet/corefx/issues/30650)
+
+### Changes
+
+* On .NET Framework, the native code portion of Microsoft.Data.SqlClient has been split into its own package, Microsoft.Data.SqlClient.SNI, which is a dependency of Microsoft.Data.SqlClient when targeting .NET Framework. This fixes the x64-only issue and adds x86 support on .NET Framework. Functionally, Microsoft.Data.SqlClient.dll is now AnyCPU (it was previously mixed-mode) and it will dynamically load x86\SNI.dll or x64\SNI.dll at runtime depending on the architecture of the underlying process. Applications targeting .NET Framework will need to ensure the SNI.dll files in their respective subfolders are deployed with the application.
+* Fixed various typos in code comments and changed ".Net" to ".NET" in comments and error messages for consistency.
+* Updated SqlClient manual test feature detection from [CoreFx PR 34040](https://github.com/dotnet/corefx/pull/34040)
+* Reduced metadata memory size in SqlClient from [CoreFx PR 34393](https://github.com/dotnet/corefx/pull/34393)
+* SqlClient add TestPacketNumberWraparound fix and test from [CoreFx PR 34709](https://github.com/dotnet/corefx/pull/34709)
+* SqlClient enhancement: enable GetFieldValue\<XmlReader\> from [CoreFx PR 34880](https://github.com/dotnet/corefx/pull/34880)
+* Allow specifying a UDT SqlParameter value as bytes from [CoreFx PR 36678](https://github.com/dotnet/corefx/pull/36678)
+
+.NET Core changes
+
+* Enable BeginExecuteReader methods on SqlCommand from [CoreFx PR 33660](https://github.com/dotnet/corefx/pull/33660)
+
+.NET Core performance improvements
+
+* Avoid unnecessary Span to ReadOnlySpan casts from [CoreFx PR 34646](https://github.com/dotnet/corefx/pull/34646)
+* SqlClient environment change optimization from [CoreFx PR 34573](https://github.com/dotnet/corefx/pull/34573)
+* Changes in 1.0.19123.2 Preview
+  * Memory allocation enhancement from [CoreFx PR 34047](https://github.com/dotnet/corefx/pull/34047)
```

Comment:
make spacing consistent

Reply:
This is done intentionally as a sub-list.